src/node_modules/app/components/AccessibilityMenu.svelte

```diff
@@ -65,23 +80,38 @@
 		<label for=''>{label}</label>
 		{#if $_currentSetting.values}
 			<div class='controlContainer'>
-				{#each $_currentSetting.values as value, index}
-					<div>
-						<label for='input-{index}'>
-							{#if showLabels}
+				{#if useRadios}
+					{#each $_currentSetting.values as value, index}
+						<div>
+							<label for='input-{index}'>
 								{$_formatValue(value)}
-							{/if}
-						</label>
-						<input
-							name='input-{index}'
-							checked={$_currentSetting.value === value}
-							class='clickable'
-							on:change={() => updateCurrentValue(value)}
-							type='radio'
-							{value}
-						>
-					</div>
-				{/each}
+							</label>
+							<input
+								name='input-{index}'
+								checked={$_currentSetting.value === value}
+								class='clickable'
+								on:change={() => updateCurrentValue(value)}
+								type='radio'
+								{value}
+							>
+						</div>
+					{/each}
+				{:else}
+					<button
+						class:clickable={hasPrevValue}
+						disabled={!hasPrevValue}
+						on:click={clickedPrev}
+					>
+						<Icon glyph={hasNumericValues ? Minus : ChevronLeft} />
+					</button>
+					<button
+						class:clickable={hasNextValue}
+						disabled={!hasNextValue}
+						on:click={clickedNext}
+					>
+						<Icon glyph={hasNumericValues ? Plus :ChevronRight} />
```

Comment:
minor, needs a blank after the `:`

```suggestion
						<Icon glyph={hasNumericValues ? Plus : ChevronRight} />
```